Marlin/example_configurations/Cartesio/Configuration_adv.h

```diff
@@ -224,6 +224,16 @@
 #define EXTRUDER_AUTO_FAN_TEMPERATURE 35
 #define EXTRUDER_AUTO_FAN_SPEED   255  // == full speed
 
+// Define a automatic fan pin to cooldown heatbed
+// 
+// When the delta temperature between current and target bed temperature is big, 
+// heatbed cooldown fan will automaticly blow to accelerate the cooldown proccess.
+// Useful when you have a PEI build surface.
+#define HEATBED_AUTO_FAN_PIN -1
```

Comment:
Quite confused on this part. I was not able to access so many kinds of motherboards but I think most 3d printers on the market don't feature a heat bed cooling fan, so this is more likely a DIY custom feature (Not associated with any specific motherboard). 
On my own MKS Gen1.4 board I need to change predefined 24V E1 MOSFET pin to FAN1_PIN to enable extra fan. If I want even more fan control I need sth. like RRD fan extender or [LED pwm control board](https://www.aliexpress.com/item/XD-93A-3W-LED-PWM-dimming-driver-support-IN-7-30V-OUT-700mA-5pcs-lot-free/32298096364.html) alike and wire any unused pwm pin to pwm signal input. Then it's not a motherboard business. 
I think editing HEATBED_AUTO_FAN_PIN in configuration_adv.h is more straight forward ( Just like CONTROLLERFAN_PIN & E_AUTO_FAN_PIN ).